app/_locales/en/messages.json

```diff
@@ -2113,6 +2113,9 @@
     "message": "This gas fee has been suggested by $1. Overriding this may cause a problem with your transaction. Please reach out to $1 if you have questions.",
     "description": "$1 represents the Dapp's origin"
   },
+  "gasFee": {
```

Comment:
I checked with the MetaMask team and for generic terms like this it may be best to leave them unscoped to a particular feature so it's easier to find and reuse in other places.